src/pages/myPage/ui/index.tsx

```
@@ -2,32 +2,42 @@ import { useNavigate } from 'react-router-dom'
 
 import styled from 'styled-components'
 
-import { Header, SvgButton, ContentHeader } from '@shared/ui'
+import { Header, SvgButton, ContentHeader, Loading, Error } from '@shared/ui'
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)